features/main_page.feature

```diff
@@ -30,6 +30,13 @@ Feature: Main Page Content and Navigation
     When I click on the title of the first game
     Then I should be redirected to the game details page
 
+
+  Scenario: Displaying links
+    Given I am on the homepage
+    Then I should see a link to view games by cost
+    And I should see a link to view latest game
+    And I should see a link to view 3D games
```

Comment:
Could we also add cucumber scenarios for when we click on each of the links? those should also be covered.